service/dps/utils/src/main/java/eu/europeana/cloud/service/dps/service/utils/validation/DpsTaskValidator.java

```diff
@@ -372,5 +402,6 @@ public String getExpectedName() {
     INPUT_DATA,
     ID,
     NAME,
-    OUTPUT_REVISION
+    OUTPUT_REVISION,
+    TARGET_INDEXING_DATABASE
```

Comment:
TARGET_INDEXING_DATABASE is just a prameter, no need for an extra DpsTaskFieldType